assets/localmetrics/service-monitor.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: lso-metrics-exporter
+  namespace: default
+  labels:
+    app: lso-metrics-exporter
+spec:
+  namespaceSelector:
+    matchNames:
+      - default
```

Comment:
why these use default namespaces?

Reply:
I understand we are replacing them later on, but we should start with right namespace to begin with.

Reply:
using correct namespace in the template.